include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -8904,10 +8913,17 @@ def err_bounds_type_annotation_lost_checking : Error<
 	"%select{assignment|increment|decrement|call|volatile}0 expression not allowed in "
 	"%select{expression|dynamic check expression|count expression|byte count expression|bounds expression}1">;
 
-  def err_checked_scope_type : Error<
+  def err_checked_scope_type_for_declaration : Error<
     "%select{parameter|return|variable|member}0 cannot have an unchecked "
     "%select{pointer|array}1 type in a checked scope or function">;
 
+  def err_checked_scope_type_for_expression : Error<
```

Comment:
There are some grammatical errors in this error message.  I suggest: "cannot use a %select{parameter|return|variable|member}0 with an unchecked type in a checked scope or function".